packages/react-components/.storybook/preview.js

```diff
@@ -3,24 +3,24 @@ import * as rootPreview from '../../../.storybook/preview';
 // load global styles
 import '../public/intro.css';
 
+/** @type {NonNullable<typeof rootPreview.parameters['options']>} */
+const options = {
```

Comment:
any extended custom logic is now extended so when running migration is more clear/visible what needs to be added back. This also fixes previously `undefined` ghost type for options being spread...